src/setup-jest.ts

```diff
@@ -2,12 +2,22 @@
 
 require('./reflect-metadata');
 
-require('zone.js/dist/zone.js');
-require('zone.js/dist/proxy.js');
-require('zone.js/dist/sync-test');
-require('zone.js/dist/async-test');
-require('zone.js/dist/fake-async-test');
-require('./zone-patch');
+try {
+  // If the user are using zone.js 0.11.1+
+  // all jest support logic are implemented inside zone.js
+  // we only need to load zone-testing.umd.js module
+  require('zone.js/bundles/zone-testing-bundle.umd.js');
+} catch (err) {
+  // Fallback logic to load zone and zone-patch
+  // when the user still use zone.js 0.10.x
+  // TODO: @JiaLiPassion, remove the fallback when Angular 10 LTS finished.
+  require('zone.js/dist/zone');
```

Comment:
I'm not sure if we can remove this fallback later because the preset supposes to support various Angular versions. If removing fallback means we don't have backward compatibility anymore for Angular < 9 ?

Reply:
Yeah, totally depends on the policy of the library, of course we can support any version of Angular with the latest version of the jest-preset-angular, or we can support different version of Angular with different versions of the jest-preset-angular. I will remove this comment.